static/_redirects

```diff
@@ -320,66 +322,109 @@ https://0-20-0.docs.pomerium.com/category/guides https://0-20-0.docs.pomerium.co
+/docs/reference/identity-provider-client-id /docs/reference/identity-provider-settings#identity-provider-client-id
+/docs/reference/identity-provider-client-secret /docs/reference/identity-provider-settings#identity-provider-client-secret
+/docs/reference/identity-provider-client-secret-file /docs/reference/identity-provider-settings#identity-provider-client-secret-file
+/docs/reference/identity-provider-name /docs/reference/identity-provider-settings#identity-provider-name
+/docs/reference/identity-provider-request-params /docs/reference/identity-provider-settings#identity-provider-request-params
+/docs/reference/identity-provider-scopes /docs/reference/identity-provider-settings#identity-provider-scopes
+/docs/reference/identity-provider-url /docs/reference/identity-provider-settings#identity-provider-url
+/docs/reference/identity-provider-refresh-directory-settings /docs/reference/identity-provider-settings#identity-provider-refresh-directory-settings
 
 # Certificates
-/docs/reference/certificate-authority /docs/reference/certificates
-/docs/reference/certificates /docs/reference/certificates
-/docs/reference/client-certificate-authority /docs/reference/certificates
-/docs/reference/client-crl /docs/reference/certificates
+/docs/reference/certificate-authority /docs/reference/certificates#certificate-authority
+/docs/reference/certificates /docs/reference/certificates#certificates
+/docs/reference/client-certificate-authority /docs/reference/certificates#client-certificate-authority
+/docs/reference/client-crl /docs/reference/certificates#client-crl
 
 # Branding
-/docs/reference/branding/darkmode-primary-color /docs/reference/branding
-/docs/reference/branding/darkmode-secondary-color /docs/reference/branding
-/docs/reference/branding/error-message-first-paragraph /docs/reference/branding
-/docs/reference/branding/favicon-url /docs/reference/branding
-/docs/reference/branding/logo-url /docs/reference/branding
-/docs/reference/branding/primary-color /docs/reference/branding
-/docs/reference/branding/secondary-color /docs/reference/branding
-
+/docs/reference/branding/darkmode-primary-color /docs/reference/branding#darkmode-primary-color
+/docs/reference/branding/darkmode-secondary-color /docs/reference/branding#darkmode-secondary-color
+/docs/reference/branding/error-message-first-paragraph /docs/reference/branding#error-message-first-paragraph
+/docs/reference/branding/favicon-url /docs/reference/branding#favicon-url
+/docs/reference/branding/logo-url /docs/reference/branding#logo-url
+/docs/reference/branding/primary-color /docs/reference/branding#primary-color
+/docs/reference/branding/secondary-color /docs/reference/branding#secondary-color
+
+# TLS
+/docs/reference/routes/tls-server-name /docs/reference/routes/tls#tls-upstream-server-name
+/docs/reference/routes/tls-upstream-server-name /docs/reference/routes/tls#tls-upstream-server-name
+/docs/reference/tls-downstream-server-name /docs/reference/routes/tls#tls-downstream-server-name
```

Comment:
I think we should keep both of the existing redirects from lines 381-382, so something like this?

```suggestion
/docs/reference/routes/tls-upstream-server-name /docs/reference/routes/tls#tls-upstream-server-name
/docs/reference/tls-upstream-server-name /docs/reference/routes/tls#tls-upstream-server-name
/docs/reference/routes/tls-downstream-server-name /docs/reference/routes/tls#tls-downstream-server-name
/docs/reference/tls-downstream-server-name /docs/reference/routes/tls#tls-downstream-server-name
```